src/content/snapshot/branching-and-baselines.md

```diff
@@ -183,12 +225,19 @@ If there is, check the status of that snapshot:
 
 The last case bears thinking about a bit. Consider this scenario:
 
-```shell
+<!-- ```shell
 x - Build N
 |
 y - Build N+1
 |
 z - Build N+2
+``` -->
+
+```mermaid
```

Comment:
This might be a mermaid feature/implementation, but I was wondering if we could remove the dotted line at the end of the graph, as users unfamiliar with it might reach out to us with questions or put up a pull request.

Reply:
The only way to remove it is by hiding the branch information. Then we won't get the branch name either.